src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -597,7 +597,7 @@ void AbortEnergyPlus()
     // Close the socket used by ExternalInterface. This call also sends the flag "-1" to the ExternalInterface,
     // indicating that E+ terminated with an error.
     if (NumExternalInterfaces > 0) CloseSocket(-1);
-    std::exit(EXIT_FAILURE);
+    return EXIT_FAILURE;
```

Comment:
I was a bit nervous about this, because of course a call to ShowFatal needs to result in the program stopping instantly, not returning and accidentally continuing.  After some browsing in the code, however, I see that this is only called in one spot, EnergyPlusPgm, and only if an exception is thrown during the simulation, and is immediately followed by the EndEnergyPlus call.  Since ShowFatal raises an exception now (instead of calling AbortEnergyPlus), this is a reasonable approach.  :+1: